cypress.json

```diff
@@ -5,6 +5,8 @@
   "reporterOptions": {
     "configFile": "reporter-config.json"
   },
+  "viewportWidth": 2000,
```

Comment:
kind of beyond typical resolution 1920x  1080
any reason, we have to set this width to pass the test cases?

Reply:
the default view point dimension is not large enough, causing the notification div cover the button, so we double the default size, to allow the notification div display without overlap other UI elements.

Reply:
Not too many customers will have ulta wide screen. Could we file a ux bug to notification team

Reply:
the error notification only shows on Cypress bundled chrome but not the normal chrome browser, so I was wondering something with Cypress instead.